addons/hr_attendance/static/src/components/greetings/greetings.xml

```diff
@@ -2,9 +2,11 @@
 <templates xml:space="preserve">
     <t t-name="hr_attendance.public_kiosk_greetings">
         <t t-if="this.attendance">
-            <t t-call="hr_attendance.EmployeeBadge">
-                <t t-set="employeeAvatar" t-value="this.employeeAvatar"/>
-            </t>
+            <div class="position-relative" style="margin: 15px;">
+                <t t-call="hr_attendance.EmployeeBadge">
+                    <t t-set="employeeAvatar" t-value="this.employeeAvatar"/>
+                </t>
+            </div>
```

Comment:
I'm afraid we can't implement this in stable, as it alters the button's XPath, potentially breaking any customizations reliant on it. I believe adding the `m-5` Bootstrap class to the button will resolve the issue.